ci/lint/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# See test/lint/README.md for usage.
+FROM ubuntu:jammy
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LC_ALL=C.UTF-8
+
+# Must be built from ./ci/lint/ for these paths to work.
+COPY ./docker-entrypoint.sh /entrypoint.sh
+COPY ./04_install.sh /install.sh
+
+RUN apt-get update && \
+  /install.sh && \
+  mv /tmp/shellcheck-*/shellcheck /usr/bin/ && \
```

Comment:
nit: Maybe just put the bin in this dir in `./ci/lint/04_install.sh` to avoid duplication here?

Reply:
Good call, will do.